src/lib/Scenes/Artwork/Components/CommercialButtons/BidButton.tsx

```diff
@@ -0,0 +1,153 @@
+import { Button, Sans } from "@artsy/palette"
+import { BidButton_artwork } from "__generated__/BidButton_artwork.graphql"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+
+export const PREDICTION_URL = "https://live-staging.artsy.net"
```

Comment:
We'll probably want this to be `live.artsy.net`. I've checked and we literally do the same thing for both:

https://github.com/artsy/eigen/blob/a677b504ccdb7fad552f7749912d2f048a925198/Artsy/App/ARSwitchBoard.m#L325-L326

But using the prod URL feels better.